packages/scss/src/commons/config.scss

```diff
@@ -85,6 +85,27 @@ $fontTokens: (
 	body-XS-fontWeight: var(--pr-t-font-fontWeight-regular),
 );
 
+$borderRadiusVars: (
+	'0': 0px,
+	'25': 2px,
+	'50': 4px,
+	'75': 6px,
+	'100': 8px,
+	'150': 12px,
+	'200': 16px,
+	'300': 24px,
+	'400': 32px,
```

Comment:
What about 1000 for 999px?

Reply:
We already have `--pr-t-border-radius-full`. 
It seems overkill imho.